handlers/HostNameHandler.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace cetver\LanguagesDispatcher\handlers;
+
+use Yii;
+use yii\base\InvalidConfigException;
+use yii\helpers\ArrayHelper;
+use yii\web\Request;
+
+/**
+ * Class HostNameHandler
```

Comment:
Make it more informative:

Class HostNameHandler handles the languages based on the host map.